src/pages/Jobs/ScheduleAgents/index.js

```diff
@@ -16,18 +16,231 @@
  51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { Spinner } from "react-bootstrap";
+import CommonFields from "../../../components/Upload/CommonFields";
+import InputContainer from "../../../components/Widgets/Input";
+import Alert from "../../../components/Widgets/Alert";
+import Button from "../../../components/Widgets/Button";
+import { getAllFolders } from "../../../services/folders";
+import { scheduleJobs } from "../../../services/upload";
+import { getBrowseData } from "../../../services/browse";
 
 const ScheduleAgents = () => {
+  const initialScheduleAnalysisData = {
+    folderId: 1,
+    uploadId: 1,
+  };
+  const initialScanFileData = {
+    analysis: {
+      bucket: true,
+      copyrightEmailAuthor: false,
+      ecc: false,
+      keyword: false,
+      mime: false,
+      monk: false,
+      nomos: false,
+      ojo: false,
+      package: false,
+    },
+    decider: {
+      nomosMonk: false,
+      bulkReused: false,
+      newScanner: false,
+      ojoDecider: false,
+    },
+    reuse: {
+      reuseUpload: 0,
+      reuseGroup: "",
+      reuseMain: false,
+      reuseEnhanced: false,
+    },
+  };
+  const initialFolderList = [
+    {
+      id: 1,
+      name: "Software Repository",
+      description: "Top Folder",
+      parent: null,
+    },
+  ];
+  const initialBrowseData = {
+    page: 1,
+    limit: 100,
+    groupName: "",
+    recursive: true,
```

Comment:
Please set recursive to false. We only need data from current folder not every folder.

Reply:
Changed recursive to false